composer.json

```diff
@@ -28,6 +28,7 @@
         "php": ">=5.6",
         "cakephp/collection": "^3.6",
         "cakephp/database": "^3.6",
+        "cakephp/datasource": "^3.6",
```

Comment:
Ideally, this is in require-dev, as latest versions show it being auto-pulled as dependency.
But for now is also fine I think :)

Reply:
Sounds like we should at some point fix the core here as well.

Reply:
Also Installing cakephp/core (3.0.0): Downloading (100%) is not 3.6 which is needed :)